### DISCUSSION TITLE
Banco do Brasil extracts Open Banking investment data with Quarkus and Kafka

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#25885] [Quarkus Blog/Website]
# Banco do Brasil extracts Open Banking investment data with Quarkus and Kafka

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/banco-do-brasil-open-banking-user-story/

**Comment by rubensdemelo** [comment]
Parabéns Felipe! Obrigado por compartilhar a experiência do BB.

**Reply by viniciusw95** [reply]
Parabéns!

**Comment by jimmmisss** [comment]
Que maravilha de case de sucesso, parabéns a todos os envolvidos.

**Comment by AndreBrust** [comment]
Parabéns Felipe, como sempre se destacando. Excelente caso de sucesso, e mais um no BB.

**Comment by fercomunello** [comment]
Parabéns aos envolvidos nessa decisão, o Quarkus está chegando cada vez mais longe! :)
